vcd_cli/role.py

```diff
@@ -21,6 +21,9 @@ def role(ctx):
             
         vcd role list_rights
             Get list of rights associated with a given role.
+        
+        vcd role create role1 description 'Disk: View Properties' 'Provider vDC: Edit' --org myOrg
```

Comment:
Can we use something like sample_role/myRole and sample_description/myDescription instead of role1 and description respectively?

Reply:
done.